.github/workflows/ks-review.yml

```diff
@@ -144,6 +144,7 @@ jobs:
           echo "K8S_SECRET_CSRF_TRUSTED_ORIGINS=.test.kuva.hel.ninja" >> $GITHUB_ENV
           echo "K8S_SECRET_LOGIN_REDIRECT_URL=${{ env.EMPLOYER_URL }}" >> $GITHUB_ENV
           echo "K8S_SECRET_LOGIN_REDIRECT_URL_FAILURE=${{ env.EMPLOYER_URL }}/login?error=true" >> $GITHUB_ENV
+          echo "K8S_SECRET_LOGOUT_REDIRECT_URL=${{ env.EMPLOYER_URL }}/login?logout=true" >> $GITHUB_ENV
```

Comment:
Do we need these changes also to `te-review.yml` and `bf-review.yml` ?

Reply:
@haavikko is this needed? if yes I can fix it now, thank you

Reply:
Good catch, I think it's needed so thta login works in review environments